openshift/dev.env

```diff
@@ -0,0 +1,9 @@
+#----- OpenShift Settings
+
+MEMORY_REQUEST=512Mi
+MEMORY_LIMIT=2Gi
+CPU_REQUEST=500m
+CPU_LIMIT=1
+VOLUME_CAPACITY=5Gi
```

Comment:
this feels really large for a dev instance. Perhaps we can scale this down to 1Gi. @caggles we are working on deploying an HA mongo instance do you have thoughts on storage provisioning when it comes to `dev-test-prod`?

Reply:
Sure. I can change it to 1 Gi or even lower if needed. I have no ideas right now about storage required for test/prod instances. It will depend a lot on the number of blogs, text, etc. And that is the reason I haven't added the test.env and prod.env files yet. My two cents: We can go low for now and take a call on that once we have better data.

Reply:
heck, even smaller an 1Gi would work for dev! the smaller you can manage, the better - especially if you plan to us a PR-based pipeline and will therefore have many instances up at once.